Backpack.podspec

```diff
@@ -16,7 +16,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "Backpack"
-  s.version          = "0.3.0"
+  s.version          = "0.4.0"
```

Comment:
This increment should be done by the release task so don't do it here

Reply:
Ah - I was 50/50 on whether I did it or the release did. I chose...poorly.

Reply:
No problem. The code is here https://github.com/Skyscanner/backpack-ios/blob/ebeb518d36250225c6bb0af9e8139a4d29ddae71/Rakefile#L75-L77